spec/site/tlc/modes_spec.rb

```diff
@@ -166,6 +166,24 @@
         switch_normal_control
       end
     end
+
+    # Verify that site goes back to normal control after M0001 timeout
+    #
+    # 1. Given the site is connected
+    # 2. Send the control command to switch to Yellow flash
+    # 3. Wait for status Yellow flash
+    # 4. Send command to switch to normal control
+    # 5. Wait for status "Yellow flash" = false, "Controller starting"= false, "Controller on"= true"
+    specify 'yellow flash can be activated with M0001', sxl: '>=1.0.7' do |example|
```

Comment:
the spec name must be changed, if this is a new test. another test already exist with this name. it should be describtive and work as a continuation of the phrase 'Traffic Light Controller Operational ...'